docs/reference/semaphore-changelog.md

```diff
@@ -4,6 +4,24 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of February 03, 2020
+
+- Additions to ubuntu1804 image:
+    - libmaxminddb0
+    - libmaxminddb-dev
+- ubuntu1804 kernel settings changes, introduced because of docker requirements:
```

Comment:
I am not an expert on this but I am wondering if "introduced because of" is appropriate language for changelog, do we have to add that part of can we go for just "ubuntu1804 kernel settings changes"?

Reply:
Removed ", introduced because of docker requirements"